build.xml

```diff
@@ -14,23 +14,23 @@
 
     <target name="build" depends="build-deploy-part-1-db-independent,build-deploy-part-2-db-dependent" description="Builds application for production preserving your DB."/>
 
-    <target name="build-demo" depends="wipe,build-version-generate,composer-prod,backend-api-oauth-keys-generate,redis-check,npm,dirs-create,assets,db-demo,grunt,error-pages-generate,warmup,product-search-recreate-structure,product-search-export-products,clean-redis-old" description="Builds application for production with clean demo DB."/>
+    <target name="build-demo" depends="wipe,build-version-generate,monorepo-backend-api-integrate,composer-prod,backend-api-oauth-keys-generate,redis-check,npm,dirs-create,assets,db-demo,grunt,error-pages-generate,warmup,product-search-recreate-structure,product-search-export-products,clean-redis-old,monorepo-backend-api-disintegrate" description="Builds application for production with clean demo DB."/>
```

Comment:
please use `shopsys_framework.build_demo`

Reply:
I just tried to do it, and when I call from command line `./phing build-demo`, it uses the phing target from the framework, and not this one... And that's not correct behavior in my opinion

Reply:
i think this tatget seems to be easy
```xml
<target name="build-demo"  description="Builds application for production with clean demo DB."
    <phingcall target="monorepo-backend-api-integrate"/>
    <phingcall target="shopsys_framework.build-demo"/>
    <phingcall target="monorepo-backend-api-disintegrate"/>
</target>
```